backend/api/Areas/Tools/Controllers/ImportController.cs

```diff
@@ -0,0 +1,96 @@
+using System;
+using System.Linq;
+using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Pims.Dal.Helpers.Extensions;
+using Entity = Pims.Dal.Entities;
+using System.Collections.Generic;
+using Pims.Api.Areas.Tools.Models;
+using Pims.Api.Models;
+using Pims.Dal.Services.Admin;
+
+namespace Pims.Api.Areas.Admin.Controllers
+{
+    /// <summary>
+    /// ImportController class, provides endpoints for managing parcels.
+    /// </summary>
+    // [Authorize (Roles = "administrator")]
+    [ApiController]
+    [Area("admin")]
+    [Route("/api/[area]/[controller]")]
+    public class ImportController : ControllerBase
+    {
+        #region Variables
+        private readonly ILogger<ImportController> _logger;
+        private readonly IPimsAdminService _pimsAdminService;
+        private readonly IMapper _mapper;
+        #endregion
+
+        #region Constructors
+        /// <summary>
+        /// Creates a new instance of a ImportController class.
+        /// </summary>
+        /// <param name="logger"></param>
+        /// <param name="pimsAdminService"></param>
+        /// <param name="mapper"></param>
+        public ImportController(ILogger<ImportController> logger, IPimsAdminService pimsAdminService, IMapper mapper)
+        {
+            _logger = logger;
+            _pimsAdminService = pimsAdminService;
+            _mapper = mapper;
+        }
+        #endregion
+
+        #region Endpoints
+        /// <summary>
+        /// POST - Add an array of new properties to the datasource.
+        /// </summary>
+        /// <param name="models">An array of property models.</param>
+        /// <returns>The properties added.</returns>
+        [HttpPost("/api/[area]/properties")]
+        public IActionResult ImportProperties([FromBody] PropertyModel[] models)
```

Comment:
I think I'm missing something here, say all of the parcels I add don't exist in the system yet. Isn't this API going to add a whole bunch of empty parcels? Am I missing where all the property data gets copied into the new parcel?

Reply:
similarly, assuming that the parcels already exist and I'm trying to update them, where does this controller overwrite the existing parcel's data?

Reply:
Same here. Could you please explain how the data is copied across?

Reply:
This class wasn't supposed to be part of this PR, but forgot to switch branches.  I'll remove for now.

Reply:
I've left it in as time is not on my side.  I'll find some time tonight to complete the endpoint as a separate PR.